keanu-python/tests/test_sampling.py

```diff
@@ -7,31 +7,35 @@
 from examples import thermometers
 from keanu import BayesNet, KeanuRandom, Model
 from keanu.algorithm import sample, generate_samples
-from keanu.vertex import Gamma, Exponential, Cauchy
+from keanu.vertex import Gamma, Exponential, Cauchy, Bernoulli
+from typing import Any
+import matplotlib
+matplotlib.use('Cairo')
```

Comment:
can you add a comment to say what this means?

Reply:
So in my local dev Mac OS X environment, python is not installed as a framework .. and this threw a `RuntimeError` when I directly imported `matplotlib.pyplot` because matplotlib's default backend `MacOSX` for Mac users doesn't work with a flat Unix-y shared library.

Not a problem in our CI though, since Python is probably correctly installed.